src/playable.cpp

```diff
@@ -0,0 +1,26 @@
+#include "rive/artboard.hpp"
+#include "rive/playable.hpp"
+
+using namespace rive;
+
+Playable::Playable(ArtboardInstance* abi) : m_ArtboardInstance(abi) {
```

Comment:
These look mostly like stubs. I'm curious why you chose to add a cpp file as opposed to stubbing them out in the header.

Reply:
Allows me to not having to include the related .hpp files that are only needed for the impl.